components/Cards/index.js

```diff
@@ -17,3 +16,113 @@
 // </div>
 //
 // Create a card for each of the articles and add the card to the DOM.
+
+const cardsContainer = document.querySelector(".cards-container");
+
+const articleData = "https://lambda-times-backend.herokuapp.com/articles"
+
+const tabs = document.querySelector(".tabs");
+
+tabs.addEventListener("click", function(event) {
+    let tabName = event.target.innerHTML;
+    let allCards = document.querySelectorAll(".card")
+    
+    const hiddenArticles = document.querySelectorAll(".hide-articles")
```

Comment:
Not clear when ".hide-articles" gets added and how it knows which articles to add this class to?